src/tests/electron/tests/framework-no-codecs.test.ts

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import * as path from 'path';
+import { createAppController } from 'tests/electron/common/create-application';
+import { AppController } from 'tests/electron/common/view-controllers/app-controller';
+import { CodecTestViewController } from 'tests/electron/common/view-controllers/codecs-test-view-controller';
+
+/*
+We bundle a mirrored version of electron with minimal
+audio/video codecs to avoid shipping proprietary codecs
+we don't need in the release build. This e2e test checks
+to ensure we have the right codecs. If this test fails,
+it's likely node_modules/electron/dist was not replaced
+with the appropriate electron mirror.
+*/
+
+const releaseTests = process.env.RUN_RELEASE_TESTS === 'true';
+(releaseTests ? describe : describe.skip)(
+    'electron bundled with minimal audio-video codecs',
```

Comment:
The test is verifying that a certain codec doesn't exist, rather than that a minimal codec does exist; maybe "without proprietary" instead of "with minimal"?